pkg/virt-controller/services/template_test.go

```diff
@@ -211,6 +211,39 @@ var _ = Describe("Template", func() {
 
 			return vmi
 		}
+
+		Context("Use emulation", func() {
+			It("should add the kvm resource when emulation is disabled", func() {
```

Comment:
Thanks for this PR, and thanks for pointing I missed it in my PR.

small suggestion to make the English of the test name more readable:

```suggestion
		Context("the kvm resource", func() {
			It("should be added when emulation is disabled", func() {
```

Reply:
These two tests cover both the KVM resource and the `--allow-emulation` command line option for virt-launcher-monitor/virt-launcher, and they are both controlled by the cluster-wide use emulation option so I prefer keeping it as is to better reflect reality.

Reply:
I tend to like the naming convention proposed by @dankenigsberg, but I don't have a strong opinion on it.
If you think it's preferred this way then it's fine by me.

Reply:
> These two tests cover both the KVM resource and the `--allow-emulation` command line option for virt-launcher-monitor/virt-launcher, and they are both controlled by the cluster-wide use emulation option so I prefer keeping it as is in order to better reflect reality.

I'm no expert, but the English guides me to believe that `UseEmulation` is not a shared context for these two tests: the first doesn't use emulation, and the second one does. They seem like two opposite contexts.
However both tests share a context in the sense that they both test for `devices.kubevirt.io/kvm`.

Reply:
> > These two tests cover both the KVM resource and the `--allow-emulation` command line option for virt-launcher-monitor/virt-launcher, and they are both controlled by the cluster-wide use emulation option so I prefer keeping it as is in order to better reflect reality.
> 
> I'm no expert, but the English guides me to believe that `UseEmulation` is not a shared context for these two tests: the first doesn't use emulation, and the second one does. They seem like two opposite contexts. However both tests share a context in the sense that they both test for `devices.kubevirt.io/kvm`.

The tests are indeed opposites, one tests the presence of the KVM resource and the absence of `allow-emulation` command line option, and the other tests the opposite.